docs/userguides/networks.md

```diff
@@ -77,7 +77,9 @@ Here is a list of all L2 network plugins supported by Ape:
 | ape-polygon       | [ApeWorX/ape-polygon](https://github.com/ApeWorX/ape-polygon)             |
 | ape-polygon-zkevm | [ApeWorX/ape-polygon-zkevm](https://github.com/ApeWorX/ape-polygon-zkevm) |
 
-**NOTE**: If you are connecting an L2 network or any other network that does not have a plugin, you can use the custom network support, which is described in the [next section](#custom-network-connection).
+```{note}
+If you are connecting an L2 network or any other network that does not have a plugin, you can use the custom network support, which is described in the [next section](#custom-network-connection).
```

Comment:
**Question**: does the [next section] like still work when in a note-block like this? I am afraid it may not

Reply:
The [next section] works when locally building the docs on my end.